src/symbols/analyzer.rs

```diff
@@ -2011,187 +2046,44 @@ impl<'a> SymbolFiller<'a> {
             ModuleDecl::ExportDecl(export_decl) => match &export_decl.decl {
               Decl::Class(n) => {
                 let id = n.ident.to_id();
-                let def_symbol_id = file_module.ensure_symbol_for_swc_id(
-                  id.clone(),
-                  SymbolDecl {
-                    kind: SymbolDeclKind::Definition(SymbolNode(
-                      SymbolNodeInner::ExportDecl(
-                        NodeRefBox::unsafe_new(
-                          &file_module.source,
-                          export_decl,
-                        ),
-                        SymbolNodeInnerExportDecl::Class(
-                          NodeRefBox::unsafe_new(&file_module.source, n),
-                        ),
-                      ),
-                    )),
-                    range: export_decl.range(),
-                  },
-                );
```

Comment:
These removals resolve the todo here: https://github.com/denoland/deno_graph/pull/320/files#diff-a05c59be6d87eb681f2159cb5c6aea202a9d11d2cc5ee9e20ebfb7bc803646f9L482-L484